pkg/addons/velero/velero.go

```diff
@@ -6,14 +6,25 @@ import (
 
 	"github.com/pkg/errors"
 	ecv1beta1 "github.com/replicatedhq/embedded-cluster/kinds/apis/v1beta1"
+	"github.com/replicatedhq/embedded-cluster/pkg/kubeutils"
 	"github.com/replicatedhq/embedded-cluster/pkg/release"
 	"github.com/replicatedhq/embedded-cluster/pkg/runtimeconfig"
 	"gopkg.in/yaml.v3"
+	"k8s.io/apimachinery/pkg/runtime"
+	jsonserializer "k8s.io/apimachinery/pkg/runtime/serializer/json"
 )
 
 type Velero struct {
 	Proxy               *ecv1beta1.ProxySpec
 	ProxyRegistryDomain string
+	HostCABundlePath    string
+
+	// DryRun is a flag to enable dry-run mode for Velero.
+	// If true, Velero will only render the helm template and additional manifests, but not install
+	// the release.
+	DryRun bool
```

Comment:
shouldn't this be a parameter passed to the addon `Install` and `Upgrade` functions instead of a field here?

Reply:
nevermind. doesn't have to be